README.md

```diff
@@ -1,5 +1,13 @@
 # Node Version Manager for PowerShell
 
+[![powershellgallery](https://img.shields.io/powershellgallery/v/nvm.svg)](https://www.powershellgallery.com/packages/nvm)
+[![downloads](https://img.shields.io/powershellgallery/dt/nvm.svg?label=downloads)](https://www.powershellgallery.com/packages/nvm)
+
+| Operating System | Build Status |
+| ---------------- | ------------ |
+| Windows | [![Build status (Windows)](https://ci.appveyor.com/api/projects/status/iuytgb4lg1wp458f/branch/master?svg=true)](https://ci.appveyor.com/project/aaronpowell/ps-nvmw/branch/master) |
+| OSX | [![Build Status (OSX)](https://travis-ci.org/aaronpowell/ps-nvmw.svg?branch=master)](https://travis-ci.org/aaronpowell/ps-nvmw) |
```

Comment:
what's the thing with PowerShell projects and huge badge tables? 😄

Reply:
Makes it clear which build is passing on which OS 😛 :trollface:

Reply:
I personally never felt an ambiguity here but that's what the left side of the badges are for ;)

![windows build](https://img.shields.io/appveyor/ci/aaronpowell/ps-nvmw/pester-tests.svg?label=windows+build) ![osx build](https://img.shields.io/travis/aaronpowell/ps-nvmw/pester-tests.svg?label=macos+build)

Reply:
Us Windows devs are a simple folk ;)

Reply:
I'm trying to make Windows cool again! 😄 

![giphy 1](https://user-images.githubusercontent.com/10532611/32543905-d17ff36e-c42b-11e7-9faa-2fa465a33109.gif)